### ISSUE TITLE
训练开始loss=0

### ISSUE THREAD

**Issue by Keyird** [828Tina/textvqa_grounding_task_qwen2.5-vl-ft#7]
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 9.701440635052094e-06, 'epoch': 0.4}
